core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/sink/VisualizationOperator.java

```diff
@@ -0,0 +1,75 @@
+package edu.uci.ics.texera.dataflow.sink;
```

Comment:
I think there should be a folder for `VisualizationOperator`. Then `BarChartSink`, `PieChartSink` and other visualization operators can be sub folders within it. (should be better organized)

Reply:
So how about creating `visualization` folder under the `sink` folder?

Reply:
yeah I think that should be good, you can put `visualizationConstants.java` etc into the folder.